src/assets/team-profiles/team.ts

```diff
@@ -1,266 +1,284 @@
+import { IMember } from '@types';
+
+import avani from './avani.jpg';
+import emmanuel from './emmanuel.jpg';
+import frederik from './frederik.jpg';
+import george from './george.jpg';
+import guillaume from './guroux.jpg';
+import harry from './harry.jpg';
+import jack from './jack.jpg';
+import jai from './jai.jpg';
+import jason from './jason.jpg';
+import luit from './luit.jpg';
+import luker from './luker.jpg';
+import maarten from './maarten.jpg';
+import mia from './mia.jpg';
+import michael from './michael.jpg';
+import petra from './petra.jpg';
+import ryon from './ryon.jpg';
+import taylor from './taylor.jpg';
+import wietze from './wietze.jpg';
+import jordan from './jordan.jpg';
+
 // Migrate content from `about.mycrypto.com`
-const TEAM_MEMBERS = [
+const TEAM_MEMBERS: IMember[] = [
   {
-    profileUrl: 'taylor.jpg',
+    profileUrl: taylor,
```

Comment:
Imported in this manner it is no longer an url.
Let's rename the attribute to reflect it's value.